packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -5,26 +5,26 @@ category: management
 
 # System Log API
 
-The Okta System Log records system events related to your organization in order to provide an audit trail that can be used to understand platform activity and to diagnose problems.
+The Okta System Log records system events that are related to your organization in order to provide an audit trail that can be used to understand platform activity and to diagnose problems.
 
-The Okta System Log API provides near real-time read-only access to your organization's system log and is the programmatic counterpart of the [System Log UI](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Reports_SysLog).
+The Okta System Log API provides near real-time, read-only access to your organization's system log and is the programmatic counterpart of the [System Log UI](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Reports_SysLog).
 
-Often the terms "event" and "log event" are used interchangeably. In the context of this API, an "event" is an occurrence of interest within the system and "log" or "log event" is the recorded fact.
+The terms "event" and "log event" are often used interchangeably. In the context of this API, an "event" is an occurrence of interest within the system, and a "log" or "log event" is the recorded fact.
 
-Notes on the System Log API:
+The System Log API, which contains much more [structured data](#logevent-object) than the [Events API](/docs/reference/api/events/#event-object), supports:
 
-* It contains much more [structured data](#logevent-object) than the [Events API](/docs/reference/api/events/#event-object).
-* It supports additional [SCIM filters](#request-parameters) and the `q` query parameter, because of the presence of more structured data than the [Events API](/docs/reference/api/events/#request-parameters).
-* Its primary supported use cases are:
-  * Event data export into a security information and event management system (SIEM).
-  * System monitoring.
-  * Development debugging.
-  * Event introspection and audit.
-* It is not intended to be used as a Database as a Service (DBaaS), or otherwise directly serve data to downstream consumers without an intermediate data store.
+* Additional [SCIM filters](#request-parameters) and the `q` query parameter, because of the presence of more structured data than the [Events API](/docs/reference/api/events/#request-parameters)
```

Comment:
I stumbled over the phrase after the comma. The way I understand it, I think it could be deleted, but I'd get that double-checked.

Reply:
You're correct it can be deleted. My understanding of the comma rule here is that you only add the comma preceding "and" and "because" if a proper sentence follows, not a phrase.